test/integration/test_integration.js

```diff
@@ -42,6 +42,24 @@ test('Valid CREATE transaction', t => {
 })
 
 
+test('Valid CREATE transaction with Threshold input', t => {
```

Comment:
This test may not be that useful. The CREATE tx fulfillment isn't reliant on any output.

I'd instead test the following:

1. CREATE tx with std ed25519 input and threshold output

and 

1. CREATE tx with std ed25519 input and threshold output
2. TRANSFER tx spending the threshold output into an ed25519 output

I'd also test:

- m-of-n
- multi signature transactions
- serialization of these transactions __during__ signing.

For last point, think about a real-life scenario. You and a friend want to spend a transaction together, so you start signing with your key. They you send the half-signed tx to your friend. They sign as well and send it off to the network.